pkg/storage/swift/swift.go

```diff
@@ -267,6 +267,13 @@ func (d *driver) ensureAuthURLHasAPIVersion() error {
 
 	path := parsedURL.Path
 
+	// There is a bug in OSP13 that generates the url path with double /
+	// i.e. http://10.1.52.2:5000//v3
+	// We have to be tolerant for this and support such a format as well.
+	if strings.HasPrefix(path, "//") {
```

Comment:
What if someone generates the URL path with triple ///, `http://10.1.52.2:5000///v3`? Or uses `http://10.1.52.2:5000/./v3`?

Reply:
as you wish :)